src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -129,12 +252,15 @@ def _get_runtime_log_level():
         if pytwin_level == PyTwinLogLevel.PYTWIN_LOG_CRITICAL:
             return LogLevel.TWIN_LOG_FATAL
 
-    def _initialize_evaluation(self, parameters: dict = None, inputs: dict = None, runtime_init: bool = True):
+    def _initialize_evaluation(
+        self, parameters: dict = None, inputs: dict = None, inputfields: dict = None, runtime_init: bool = True
+    ):
         """
         Initialize the twin model evaluation with dictionaries:
         (1) Initialize parameters and/or inputs values to their start values (default values found in the twin file).
-        (2) Update parameters and/or inputs values with provided dictionaries. Ignore values whose names are not found
-        in the list of parameters/inputs names of the twin model. (Value is kept to the default value in this case.)
+        (2) Update parameters and/or inputs values with provided dictionaries (including input field snapshot if
+        applicable. Ignore values whose names are not found in the list of parameters/inputs names of the twin model.
+        (Value is kept to the default value in this case.)
         (3) Initialize evaluation time to 0.
         (4) Save universal time (time since epoch) at which the method is called.
         (5) Evaluate twin model at time instance 0. Store its results into an outputs dictionary.
```

Comment:
```suggestion
        #. Update parameters and/or inputs values with provided dictionaries (including input field snapshot if
           applicable). Ignore values whose names are not found in the list of parameter/input names of the twin model.
           (The value is kept to the default value in this case.)
        #. Initialize the evaluation time to 0.
        #. Save the universal time (time since epoch) at which the method is called.
        #. Evaluate the twin model at time instance 0. Store its results into an outputs dictionary.
        
```